src/components/elements/input/MinutesDurationInput.tsx

```diff
@@ -90,6 +92,8 @@ const MinutesDurationInput = ({
           helperText={null}
           max={59}
           fullWidth={false}
+          inputWidth={105}
+          placeholder='0'
```

Comment:
Some very minor style changes on this component.  Fixed width feels like an anti-pattern but it bothered me how wide the input space was, when we know it's only 2 chars max. I tested this on mobile and it looks fine, so I think its OK.

<img width="253" alt="Screenshot 2024-02-22 at 3 16 10 PM" src="https://github.com/greenriver/hmis-frontend/assets/5333982/0743abe2-311a-4f0a-af0d-b84d2a93445a">